vignettes/Demography.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: "Demography"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Demography}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+suppressPackageStartupMessages(library(ggplot2))
```

Comment:
ggplot??!?!?!?! yes!

Reply:
😄 we had ggplot all along in DESCRIPTION! Not the whole of tidyverse though. I could remove it? 😛